plugins/module_utils/nbdkit/nbdkit.go

```diff
@@ -41,6 +41,9 @@ type NbdkitConfig struct {
 	VmName      string
 	Compression string
 	UUID        string
+	Port        string
+	Timeout     int
```

Comment:
Here, As I said in the module migrate.go, I dont think we should expose to the user the Timeout for nbdkit and the retry as well. It could be configurable at the package level, but I think it can be risky to expose those settings for several reasons (we know those retry limits are sane even in a stressed environment).